FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -25,7 +25,13 @@ struct Dashboard: View {
                 LocationHeader(location: store.state.location, fetchingContacts: store.state.fetchingContacts)
                     .padding(.bottom, 10)
                     .onTapGesture {
-                        showLocationSheet.toggle()
+                        if #available(iOS 17.0, *) {
```

Comment:
I started going down a route to try and track whether the popover tip is showing (or at the very least eligible to be showing), but it got tricky due to it being restricted to iOS 17+ so I couldn't just store the tip in the store state.

Reply:
There isn't a a hugely noticeable delay when tapping the button, so hopefully this is reasonable.